src/general/OptionsMenu.cs

```diff
@@ -59,6 +59,15 @@ public class OptionsMenu : Control
     [Export]
     public NodePath ChromaticAberrationTogglePath;
 
+    [Export]
+    public NodePath CurrentGpuPath;
+
+    [Export]
+    public NodePath VideoMemPath;
+
+    [Export]
+    public NodePath DisplayDriverPath;
```

Comment:
This should be named `DisplayRendererPath`, needs to be corrected down the line too for occurrences of "drivers".